tasks/check-stemcell-versions/main.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"log"
+	"os"
+	"path/filepath"
+
+	"github.com/cloudfoundry/runtime-ci/task-libs/bosh"
+)
+
+func main() {
+	buildDir := os.Args[1]
+	branchToCompare := os.Getenv("BRANCH_TO_COMPARE")
```

Comment:
Why read from an environment variable rather than pass an another arg?

Reply:
No major reason; I will accommodate and make use of the argument rather than the environment variable.